data/card_catalogue/unique-1.json

```diff
@@ -179,36 +129,6 @@
       "image": "assets/core/card/illustration/Prison.png"
     }
   },
-  {
-    "title": "The Swarm",
-    "upgrade": "",
-    "tier": "green",
-    "details": {
-      "rules": "Pick a small animal or insect for your target to be swarmed by.",
-      "flavour": "The sky went dark, and all sound was drowned out by the hum of their arrival.",
-      "image": "assets/core/card/illustration/The Swarm.png"
-    }
-  },
-  {
-    "title": "Glass Cannon",
-    "upgrade": "",
-    "tier": "red",
-    "details": {
-      "rules": "Until your next turn, deal double damage and take double damage.",
```

Comment:
I couldn't figure out a way to make this card make sense without a reference to 'turns', so I cut it for now.

I think we can achieve the same fantasy but with a different effect